code/game/mecha/equipment/tools/sec_tools.dm

```diff
@@ -1,15 +1,13 @@
-#define MECH_JAIL_TIME 10
-
 /obj/item/mecha_parts/mecha_equipment/tool/jail
-	name = "\improper Mounted Jail Cell"
-	desc = "Mounted Jail Cell, capable of holding up to two prisoners for a limited time. (Can be attached to Gygax)"
+	name = "mounted jail cell"
```

Comment:
Don't do this, all mecha equipment is capitalised and you shouldn't ruin that consistency, especially since it's not really relevant to the balance changes.